### PR TITLE
Order stop fields

### LINKED ISSUE #54: Order Stop Fields according to Field Guide
Requested during the May 31st PVTA meeting

### PR DESCRIPTION
**Pull request by jforgue**
Closes #54

### CODE REVIEW

**Comment by jforgue** [comment]
I didn't add any missing fields or remove unnecessary fields from the field guid here via migration. However, I did opt not to use the extra field in the view.

**Comment by dfaulken** [comment]
May we see the final product in action?

File: app/models/bus_stop.rb [hunk 1]
```diff
@@ -1,48 +1,91 @@
...
+    shelter_options: SHELTER_OPTIONS,
+    amenities: AMENITIES,
+    accessibility: ACCESSIBILITY,
+    security_and_safety: SECURITY_AND_SAFETY,
+    technology: TECHNOLOGY
+  }.freeze
+
+  SUPER_HASH.each do |hash|
+    hash.each do |name, options|
+      if options.kind_of?(Array)
```
Comment (dfaulken) [inline app/models/bus_stop.rb]: Would not `is_a?` work here?

Comment (jforgue) [reply app/models/bus_stop.rb]: It would, is that preferred?

**Comment by jforgue** [comment]
screen shot 2018-06-07 at 10 57 14 am

**Comment by dfaulken** [comment]
Cool. We can quibble about styling more when we're switching to the long term solution.

File: app/models/bus_stop.rb (hunk 1 above)
Comment (dfaulken) [reply app/models/bus_stop.rb]: I guess if it quacks like an array, we can use it.

Comment (werebus) [reply app/models/bus_stop.rb]: Of course, if you really wanted to take duck-typing to its extreme, you would do:

```ruby
if options.respond_to? :include?
```

**Pull request merged**